### DISCUSSION TITLE
names of cities in russian 375k?

### DISCUSSION THREAD

**Discussion by mb6ockatf** [monkeytypegame/monkeytype#4893] [Questions] [answered]
sometimes there are names of cities in russian 375k mode. is this normal? in english 450k there's nothing like that. also, russian 375k is impossible to use, as there're loads of words that are english but transliterated into russian, which is just cringe. For instance, there was a word "лайф", which is "life" but written in russian manner. it has nothing with russian book language, nor with real life -- none speaks like that. i'd like to suggest some corrections to it, if someone can prompt me where do you guys store wordlists, or where do you get words from?

**Comment by fehmer** [comment, marked as answer]
-  https://github.com/monkeytypegame/monkeytype/blob/master/frontend/static/languages/russian.json
- https://github.com/monkeytypegame/monkeytype/blob/master/frontend/static/languages/russian_10k.json
- https://github.com/monkeytypegame/monkeytype/blob/master/frontend/static/languages/russian_1k.json
- https://github.com/monkeytypegame/monkeytype/blob/master/frontend/static/languages/russian_25k.json
- https://github.com/monkeytypegame/monkeytype/blob/master/frontend/static/languages/russian_375k.json
- https://github.com/monkeytypegame/monkeytype/blob/master/frontend/static/languages/russian_50k.json
